core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/rules/auth/GroupsRule.scala

```diff
@@ -35,7 +35,9 @@ import tech.beshu.ror.accesscontrol.matchers.GenericPatternMatcher
 import tech.beshu.ror.implicits.*
 import tech.beshu.ror.utils.uniquelist.{UniqueList, UniqueNonEmptyList}
 
-abstract class BaseGroupsRule[GL <: GroupsLogic](val settings: Settings[GL])
+class GroupsRule[GL <: GroupsLogic](override val name: Rule.Name,
```

Comment:
Yes, it could be done like that. But we'd rather have a class per rule even if the class is trivial. I see the value in this approach.

Reply:
Ok, will restore classes per rule.

https://github.com/beshu-tech/readonlyrest-docs/pull/246 <= docs, I can't assign reviewer there

Reply:
Done, restored classes per rule.